.eslintrc.json

```diff
@@ -1,11 +1,26 @@
 {
-  "extends": "next/core-web-vitals",
-  "ignorePatterns": [
-    "*.json",
-    "**/*.json",
-    "next-env.d.ts",
-    "node_modules",
-    "dist",
-    "src/components/constants/**/*.js"
-  ]
-}
\ No newline at end of file
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": [
+        "eslint:recommended",
+        "plugin:react/recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
```

Comment:
Will this work well with prettire?

Reply:
I didn't find any issues with this and prettier.